static/app/components/events/interfaces/spans/index.tsx

```diff
@@ -58,29 +61,74 @@ class SpansInterface extends Component<Props, State> {
 
   renderTraceErrorsAlert({
     isLoading,
-    numOfErrors,
+    errors,
+    parsedTrace,
   }: {
     isLoading: boolean;
-    numOfErrors: number;
+    errors: TraceError[] | undefined;
+    parsedTrace: ParsedTraceType;
   }) {
     if (isLoading) {
       return null;
     }
 
-    if (numOfErrors === 0) {
+    if (!errors || errors.length <= 0) {
       return null;
     }
 
     const label = tn(
       'There is an error event associated with this transaction event.',
       `There are %s error events associated with this transaction event.`,
-      numOfErrors
+      errors.length
     );
 
+    const errorsMap: {
+      [spanId: string]: {operation: string; errorsCount: number};
+    } = {};
+
+    errors.forEach(error => {
+      if (!errorsMap[error.span]) {
+        if (parsedTrace.rootSpanID === error.span) {
+          errorsMap[error.span] = {
+            operation: parsedTrace.op,
+            errorsCount: 0,
+          };
+        } else {
+          const span = parsedTrace.spans.find(s => s.span_id === error.span);
+          if (!span?.op) {
+            return;
+          }
+
+          errorsMap[error.span] = {
+            operation: span.op,
+            errorsCount: 0,
+          };
+        }
+      }
+
+      errorsMap[error.span].errorsCount++;
+    });
+
     return (
       <AlertContainer>
         <Alert type="error" icon={<IconWarning size="md" />}>
-          {label}
+          <ErrorLabel>{label}</ErrorLabel>
+          <AnchorLinkManager.Consumer>
+            {({scrollToHash}) => (
+              <List symbol="bullet">
+                {Object.entries(errorsMap).map(([spanId, {operation, errorsCount}]) => (
+                  <ListItem key={spanId}>
+                    {tn('%s error in ', '%s errors in ', errorsCount)}
```

Comment:
You should actually chain tct with tn here. 
Roughly:
```js
{tct('[errors] in [link]', {
  errors: tn('%s error', '%s errors', errorsCount),
  link: <ErrorLink ...
```

Reply:
Changed in 5cdb684